pkg/kubectl/cmd/wait/wait.go

```diff
@@ -449,3 +451,7 @@ func (w ConditionalWait) isConditionMet(event watch.Event) (bool, error) {
 	obj := event.Object.(*unstructured.Unstructured)
 	return w.checkCondition(obj)
 }
+
+func extendErrWaitTimeout(err error, resource string) error {
```

Comment:
it might be worth including the resource kind as well: `timed out waiting on pod/foo-bar` for example

Reply:
Sounds good